iset.mm

```diff
@@ -72414,6 +72431,21 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
       GVSVCVPDABVBVOEABVAFABQRRVFVRDABVEEABQRSUEDEFACUFDEFBCUFUG $.
   $}
 
+  ${
+    $d A x y z $.  $d R x y z $.
+    $( Tight apartness in terms of apartness.  (Contributed by Jim Kingdon,
+       16-Feb-2025.) $)
+    dftapap $p |- ( R TAp A <-> ( R Ap A /\
+        A. x e. A A. y e. A ( -. x R y -> x = y ) ) ) $=
```

Comment:
This should be the definition.

Reply:
Uh, OK, I can change this.

In principle I am pretty indfifferent to what is "the" definition and what can be proved as an alternate definition in the form of an equivalence as a theorem. So I guess in practice I am a bit of a pushover if people want me to rearrange it.

Reply:
Done.